src/graph/validator/AdminJobValidator.cpp

```diff
@@ -49,6 +50,10 @@ Status AdminJobValidator::validateImpl() {
         }
       }
     }
+  } else if (needAppendChosenSpace()) {
```

Comment:
Based on code in repo and this PR, it seems `requireSpace` should true for all cases? Maybe just modify that one instead of add an extra `needAppendChosenSpace`?

Reply:
Already removed, no else will use,  do it here.

Reply:
I mean the `requireSpace` could be removed (every operation need space for now), use the new one is enough.

Reply:
Since all other AdminJobOp types need space name (except some cmds in ADD)... I think this would be better.
```
if (sentence_->getOp() == meta::cpp2::AdminJobOp::ADD) {
  ...
} else {
  sentence_->addPara(qctx()->rctx()->session()->space().name);
}
```